frontend/src/constants/Constants.js

```diff
@@ -8,7 +8,7 @@ export const BAD_GATEWAY_ERROR = 'badGateway';
  * @constant
  * @type {string} Type of connectionError when no error is present (internet down)
  */
-export const NO_CONNECTION_ERROR = 'noConnection';
+export const NO_CONNECTION_ERROR = 'noStatus';
```

Comment:
i find `noStatus` misleading. Why not calling it `offline` or something?

Reply:
also, wondering, wouldn't be better to move such constants to some OfflineActionTypes.js file?

Reply:
<img width="675" alt="Screenshot 2021-12-08 at 11 54 51" src="https://user-images.githubusercontent.com/1708561/145187959-9f142bdf-84e3-4854-8c20-3446e73bcf37.png">
because this is how that key comes from the BE  and to be able to use it according to the errorType 
![Screenshot 2021-12-08 at 11 59 24](https://user-images.githubusercontent.com/1708561/145188382-c88ef65c-c31d-47d6-abb3-b65f7d2ea1f0.png)

Reply:
add a comment about this.... because:
* it's not obvious
* would prevent next developer to break it